mod-credits/src/main/java/gg/solarmc/loader/credits/Credits.java

```diff
@@ -99,4 +98,16 @@ public DepositResult depositBalance(Transaction transaction, BigDecimal depositA
 		return new DepositResult(newBalance);
 	}
 
+	/**
+	 * Sets the user's account balance. Cannot fail
+	 * @param transaction the transaction
+	 * @param newAmount amount to set to
+	 */
+	public void setBalance(Transaction transaction, BigDecimal newAmount) {
```

Comment:
Don't we want *any* kind of check that `newAmount` is non-negative? We kind of rely on the invariant that it is non-negative elsewhere

Reply:
Done